### PR TITLE
[gdbm] Update documentation

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Signed-off-by: Gavin Didrichsen <gavin.didrichsen@gmail.com>

### CODE REVIEW

File: controls/gdbm_exists.rb [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
     its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
   end
 
-  command_relative_path = input('command_relative_path', value: '/bin/gdbmtool')
-  command_full_path = plan_installation_directory.stdout.strip + "#{command_relative_path}"
+  command_relative_path = input('command_relative_path', value: 'bin/gdbmtool')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, command_relative_path)
   describe file(command_full_path) do
     it { should exist }
```
Comment (MindNumbing) [inline controls/gdbm_exists.rb:22]: Can you add a check that gdbmtool is an executable?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,78 @@
...
+» Installing core/gdbm
+☁ Determining latest version of core/gdbm in the 'stable' channel
+→ Using core/gdbm/1.18.1/20200305234038
+★ Install of core/gdbm/1.18.1/20200305234038 complete with 0 new packages installed.
+» Binlinking gdbm_dump from core/gdbm/1.18.1/20200305234038 into /bin
+★ Binlinked gdbm_dump from core/gdbm/1.18.1/20200305234038 to /bin/gdbm_dump
+» Binlinking gdbm_load from core/gdbm/1.18.1/20200305234038 into /bin
+★ Binlinked gdbm_load from core/gdbm/1.18.1/20200305234038 to /bin/gdbm_load
+» Binlinking gdbmtool from core/gdbm/1.18.1/20200305234038 into /bin
+★ Binlinked gdbmtool from core/gdbm/1.18.1/20200305234038 to /bin/gdbmtool
```
Comment (MindNumbing) [inline README.md:56]: I don't see controls to test gdbm_dump / gdbm_load, is there an issue with testing these? If so can you raise an issue that the tests should be looked again?

Comment (gavindidrichsen) [reply README.md:56]: As discussed, leaving the single binary check as-is.  Thanks for the suggestion.

Comment (MindNumbing) [reply README.md:56]: No worries! SG

File: controls/gdbm_exists.rb (hunk 1 above)
Comment (gavindidrichsen) [reply controls/gdbm_exists.rb:22]: Done.  Thanks for the suggestion @MindNumbing.

**Pull request merged**